src/database/FirehoseConnector.js

```diff
@@ -187,6 +190,7 @@ class FirehoseConnector {
             meetingUniqueId,
             endpointId,
             sessionDuration,
+            conferenceDuration,
```

Comment:
I know this isn't necessarily consistent with the rest of our fields, but would it make sense to keep the Ms at the end so it's more readable?